src/table/settings/gui_settings.ini

```diff
@@ -585,6 +585,7 @@ interval = 1
 str      = STR_CONFIG_SETTING_STOP_LOCATION
 strhelp  = STR_CONFIG_SETTING_STOP_LOCATION_HELPTEXT
 strval   = STR_CONFIG_SETTING_STOP_LOCATION_NEAR_END
+post_cb  = [](auto) { MarkWholeScreenDirty(); }
```

Comment:
Isn't `SetWindowClassesDirty(WC_VEHICLE_ORDERS)` more to the point? After all, only those parts of the UI need to be updated, not the rest of the screen. Especially when there are no orders shown.